src/Tribe/Widget/Widget_Abstract.php

```diff
@@ -198,7 +222,53 @@ public function get_validated_arguments_map() {
 	/**
 	 * {@inheritDoc}
 	 */
-	public function get_arguments() {
+	public abstract function get_admin_fields();
+
+	/**
+	 * {@inheritDoc}
+	 */
+	public function filter_updated_instance( $updated_instance ) {
+		/**
+		 * Applies a filter to updated instance of a widget.
+		 *
+		 * @since TBD
+		 *
+		 * @param array<string,mixed> $updated_instance The updated instance of the widget.
+		 * @param static              $instance  The widget instance we are dealing with.
+		 */
+		$updated_instance = apply_filters( 'tribe_widget_updated_instance', $updated_instance, $this );
+
+		$registration_slug = $this->get_registration_slug();
+
+		/**
+		 * Applies a filter to updated instance of a widget arguments based on the registration slug of the widget.
+		 *
+		 * @since TBD
+		 *
+		 * @param array<string,mixed> $updated_instance The updated instance of the widget.
+		 * @param static              $instance  The widget instance we are dealing with.
+		 */
+		$updated_instance = apply_filters( "tribe_widget_{$registration_slug}_arguments", $updated_instance, $this );
+
+		return $updated_instance;
+	}
+
+	/**
+	 * {@inheritDoc}
+	 */
+	public function get_arguments( array $instance = [] ) {
+
+		$arguments = $this->arguments;
+
+		$arguments = wp_parse_args(
+			$arguments,
+			$this->get_default_arguments()
+		);
+
+		$arguments = wp_parse_args(
+			$instance,
+			$arguments
+		);
 
 		return $this->filter_arguments( $this->arguments );
```

Comment:
The calls above are preparing the `$arguments` variable against the defaults and the instance, but it's not the value that is being sent to the filtering function.

```suggestion
		return $this->filter_arguments( $arguments );
```